ecommerce_api_client/tests/test_client.py

```diff
@@ -20,7 +21,7 @@ class EcommerceApiClientTests(TestCase):
 
     def test_valid_configuration(self):
         """ The constructor should return successfully if all arguments are valid. """
```

Comment:
I'd suggest using `ddt` here to confirm that the constructor returns successfully for all valid configurations (i.e., no email or full name, email but no full name, full name but no email, both email and full name).